.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+  # pull_request event is required only for autolabeler
+  pull_request:
+    # Only following types are handled by the action, but one can default to all as well
+    types: [opened, reopened, synchronize]
```

Comment:
Is this required for it to work?

Reply:
I don't know, would like to merge on master to try

Reply:
removed